sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -1,126 +1,132 @@
+<!-- eslint-disable vue/valid-v-slot -->
 <template>
+  <v-card class="mb-3">
+    <v-container>
+      <p class="label-bold-normal mb-0 text-body-2">
+        This feature can be used to ignore specific restrictions for specific
+        applications. Restrictions can be ignored for one, or all, disbursements
+        associated with an application. If a restriction is ignored, it's effect
+        will be suspended. For example, if a restriction typically prevents an
+        eCert from getting produced and sent to NSLSC 5 days prior to study
+        start dates, then ignoring that restriction will enable the eCert to get
+        produced and sent to NSLSC, thereby 'bypassing' the restriction. Please
+        see training material for additional information.
+      </p>
+    </v-container>
+  </v-card>
   <v-card>
     <v-container>
-      <body-header title="History of Bypassed Restrictions" class="m-1">
-        <template #actions>
-          <check-permission-role :role="Role.StudentAddRestriction">
-            <template #="{ notAllowed }">
-              <v-btn
-                @click="addBypassedRestriction"
-                class="float-right"
-                color="primary"
-                data-cy="addRestrictionButton"
-                prepend-icon="fa:fa fa-plus-circle"
-                :disabled="notAllowed"
-                >Bypass A Restriction</v-btn
-              >
-            </template>
-          </check-permission-role>
-        </template>
-      </body-header>
+      <body-header title="History of Bypassed Restrictions" class="m-1" />
       <content-group class="mt-4">
         <toggle-content
-          :toggled="!bypassedRestrictions.length"
-          message="No bypassed restrictions found."
+          :toggled="!bypassedRestrictions.bypasses.length || isDraftApplication"
+          :message="getToggleContentMessage()"
         >
-          <DataTable
-            :value="bypassedRestrictions"
-            :paginator="true"
-            :rows="DEFAULT_PAGE_LIMIT"
-            :rowsPerPageOptions="PAGINATION_LIST"
-            :totalRecords="bypassedRestrictions.length"
+          <v-data-table
+            :headers="ApplicationRestrictionManagementHeaders"
+            :items="bypassedRestrictions.bypasses"
+            :items-per-page="DEFAULT_PAGE_LIMIT"
```

Comment:
I believe the following needs to be added as well:

`:items-per-page-options="ITEMS_PER_PAGE"`

Reply:
Added.